stable/luigi/values.yaml

```diff
@@ -0,0 +1,86 @@
+# Default values for luigi.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicaCount: 1
+image:
+  repository: getpolymorph/luigi
```

Comment:
Can you us get some more info on this image. I noticed it has only one tag and I cannot find a repo with the dockerfile or other build elements. If someone wants to modify the image themselves or build the image with their modifications there is no direction. Would you be able to post a source repository to docker hub for the image?